editor/project_converter_3_to_4.cpp

```diff
@@ -3236,6 +3355,22 @@ void ProjectConverter3To4::rename_gdscript_keywords(String &file_content) {
 		CRASH_COND(!reg_second.is_valid());
 		file_content = reg_second.sub(file_content, "@rpc(any_peer, call_local) func", true);
 	}
+	{
+		RegEx reg_first = RegEx("([\n]+)sync func");
```

Comment:
This regex and also `slave func` is not tested, but would be good to add tests to them.